src/styles/_colors.scss

```diff
@@ -0,0 +1,2 @@
+$login-button: #B3A6F9;
+$buttons: #5B4CCC;
```

Comment:
These aren't good names for colors.
Use color names not element based names.
Use this page: https://chir.ag/projects/name-that-color/